src/strawberry_sqlalchemy_mapper/mapper.py

```diff
@@ -151,6 +151,8 @@ def __init__(
         extra_sqlalchemy_type_to_strawberry_type_map: Optional[
             Mapping[Type[TypeEngine], Type[Any]]
         ] = None,
+        edge_type: Type = None,
```

Comment:
Added Erik to comment on this, but we should probably at the very least have a Protocol that these types should follow